packages/yourloops/test/lib/auth/api.test.tsx

```diff
@@ -28,15 +28,76 @@
 
 import { Preferences, Profile, Settings } from '../../../models/user'
 import UserApi from '../../../lib/auth/user-api'
-import HttpService from '../../../services/http'
-import { AxiosResponse } from 'axios'
+import HttpService, { ErrorMessageStatus } from '../../../services/http'
+import { AxiosResponse, AxiosResponseHeaders } from 'axios'
+import { HttpHeaderKeys } from '../../../models/api'
 
 describe('Auth API', () => {
-  const fetchMock = jest.fn()
+  const userId = 'userId'
 
-  afterEach(() => {
-    fetchMock.mockReset()
-    delete global.fetch
+  describe('getShorelineAccessToken', () => {
+    it('should get an access token from shoreline service', async () => {
+      const token = 'session-token'
+      const data = { token, userid: userId }
+      const headers = { [HttpHeaderKeys.sessionToken]: token } as AxiosResponseHeaders
+      jest.spyOn(HttpService, 'get').mockResolvedValueOnce({
+        data,
+        headers
+      } as AxiosResponse)
+
+      const response = await UserApi.getShorelineAccessToken()
+      const expectedResponse = { token, id: data.userid }
+      expect(response).toEqual(expectedResponse)
+      expect(HttpService.get).toHaveBeenCalledWith({ url: 'auth/login' })
+    })
+  })
+
+  describe('getProfile', () => {
+    it('should get the user profile', async () => {
+      const data = { firstName: 'Bernard', lastName: 'Tichaut' } as Profile
+      jest.spyOn(HttpService, 'get').mockResolvedValueOnce({ data } as AxiosResponse)
+      const response = await UserApi.getProfile(userId)
+      expect(response).toEqual(data)
+      expect(HttpService.get).toHaveBeenCalledWith({ url: `/metadata/${userId}/profile` })
+    })
+
+    it('should return undefined if profile doesn\'t exists', async () => {
+      jest.spyOn(HttpService, 'get').mockRejectedValueOnce(Error(ErrorMessageStatus.NotFound))
```

Comment:
```suggestion
      jest.spyOn(HttpService, 'get').mockRejectedValueOnce(Error('This error was thrown by a mock on purpose'))
```
Having an explicit error message like this helps debugging and also give more information about the console polluting (to know if there is some stuff that should be fixed or if it is normal)